docs/integrations/sources/sftp-bulk.md

```diff
@@ -113,6 +113,12 @@ For example, assuming your folder path is not set in the connector configuration
 
 If your files are in a folder, include the folder in your glob pattern, like `my_folder/my_prefix_*.csv`.
 
+#### Copy Raw Files Configuration
```

Comment:
This is where we should be a bit more specific (beyond this can go in the centralized file transfer docs page):
* Requires Platform V1.2 or greater
* Currently only supports Destination S3, with more to come soon.
* Currently supports files up to 1 GB in size.

Reply:
@Hesperide - Added. Can you re-review? And can you please suggest inline edits if you have further suggestions?